### PR TITLE
11.0 mig product variant sale price

### PR DESCRIPTION
**Pull request by SimoRubi**
Standard migration

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by SimoRubi** [comment]
@pedrobaeza fixed up transbot commits
squashed commits from same authors.
I don't know why Github is mixing up commits but my `git log --oneline --reverse` shows them in the correct order:
```
795edc4 [ADD][8.0] product_variant_sale_price: New module
6ef5048 Apply some PEP8 cleanup
b357ccb [NEW][8.0] product_variant_sale_price: Add test, fixes and translation
85e2f8a [FIX][8.0] product_variant_sale_price: fix E121 continuation line under-indented for hanging indent
829fe3b OCA Transbot updated translations from Transifex
39193dc [product_variant_sale_price] Also set fix_price to variant when template is created
1dc54b3 OCA Transbot updated translations from Transifex
efb9d42 [FIX] product_variant_sale_price: avoid to update list price of all variants
eafb7f3 OCA Transbot updated translations from Transifex
8a6252b [MIG] product_variant_sale_price: Migration to 11.0
```

Now I'm dealing with failing tests

**Comment by pedrobaeza** [comment]
> I don't know why Github is mixing up commits but my git log --oneline --reverse shows them in the correct order:

Yeah, I have observed that also. I don't know which order it takes.

**Comment by SimoRubi** [comment]
Increasing test coverage

**Review by tafaRU — APPROVED** [review]
I also had some tests on http://3337894-103-9af494.runbot1.odoo-community.org and it seems to work.

**Review by hurrinico — APPROVED** [review]
👍

File: product_variant_sale_price/README.rst
Comment (rafaelbn) [inline product_variant_sale_price/README.rst]: Please, just Tecnativa without "S.L." Thanks!

File: product_variant_sale_price/__manifest__.py
Comment (rafaelbn) [inline product_variant_sale_price/__manifest__.py]: Please, just Tecnativa without "S.L." Thanks!

File: product_variant_sale_price/__manifest__.py
Comment (rafaelbn) [inline product_variant_sale_price/__manifest__.py:9]: This should be https://github.com/OCA/product-variant
Take a look in: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

**Review by rafaelbn — APPROVED** [review]
Tested in runbot functionally :+1: 

Just some minor comments, please attend them before merging, thanks!

**Pull request merged**
